### PR TITLE
Add instructions for Vagrant Share

### PR DESCRIPTION
**Pull request by BronsonQuick**
Add docs for #465

### CODE REVIEW

File: docs/guides.rst [hunk 1]
```diff
@@ -83,3 +83,37 @@ In normal circumstances editing Chassis' own ``wp-config.php`` file is discourag
...
+   Move ngrok: `mv /path/to/ngrok /usr/local/bin/ngrok`
+
+2. **Run Vagrant Share**
+
+   You now need to run `vagrant share --http=vagrant.local:80`. If you're using a custom domain then you will need to use that e.g. `vagrant share --http=<your-custom-domain>.local:80`.
+   You will then have a temporary URL generated for you. e.g. `http://bea1a5c2.ngrok.io`. You need to leave this running in the background.
+
+3. **Alter your Chassis config**
+
+   Alter one of your Chassis config files to include the additional URL without the `http://`. e.g.
```
Comment (rmccue) [inline docs/guides.rst]: This should probably just be `include the additional domain`

File: docs/guides.rst [hunk 2]
```diff
@@ -83,3 +83,37 @@ In normal circumstances editing Chassis' own ``wp-config.php`` file is discourag
...
+
+   Vagrant Share requires `ngrok` to be installed on the host machine.
+   You can verify if this is installed by running `which ngrok` in a terminal. If there is no output then you will need to download and install `ngrok`_.
+   Once you've downloaded `ngrok` unzip it: `unzip /path/to/ngrok-stable-darwin-amd64.zip`
+   Move ngrok: `mv /path/to/ngrok /usr/local/bin/ngrok`
+
+2. **Run Vagrant Share**
+
+   You now need to run `vagrant share --http=vagrant.local:80`. If you're using a custom domain then you will need to use that e.g. `vagrant share --http=<your-custom-domain>.local:80`.
+   You will then have a temporary URL generated for you. e.g. `http://bea1a5c2.ngrok.io`. You need to leave this running in the background.
```
Comment (rmccue) [inline docs/guides.rst]: Could use a placeholder name here (e.g. `http://<id>.ngrok.io`) instead to make it clearer.

File: docs/guides.rst [hunk 3]
```diff
@@ -83,3 +83,37 @@ In normal circumstances editing Chassis' own ``wp-config.php`` file is discourag
...
+
+   hosts:
+       - vagrant.local
+       - bea1a5c2.ngrok.io
+
+4. **Reprovision Vagrant**
+
+   You now need to run `vagrant provision` and you should be able to access your Chassis box with the URL generated in Step 2.
+   If you see an error when you run `vagrant provision` then try running it again.
+   If you're still having trouble accessing the URL try a `vagrant reload` after you've run a successful `vagrant provision`.
```
Comment (rmccue) [inline docs/guides.rst]: Maybe move these lines to a debugging subsection below?

**Comment by BronsonQuick** [comment]
@rmccue Thank you Sir 🙇🏻‍♂️ I've added those changes.

**Pull request merged**